scripts/umbrel-os/external-storage/monitor

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# Load monitor-lib
+check_dependencies () {
+  for cmd in "$@"; do
+    if ! command -v $cmd >/dev/null 2>&1; then
+      echo "This script requires \"${cmd}\" to be installed"
+      exit 1
+    fi
+  done
+}
+
+check_dependencies readlink dirname
+UMBREL_ROOT="$(dirname $(readlink -f "${BASH_SOURCE[0]}"))/../../.."
+
+source "${UMBREL_ROOT}/scripts/monitor-lib"
+
+
+# Monitor external mount
+block_device="${1}"
+MOUNT_POINT="${2}"
+
+check_mount () {
+    # Check if Umbrel root is still bind mounted
+    if ! df -h "${UMBREL_ROOT}" | grep --quiet '/dev/sd'; then
```

Comment:
If you're passing in the block device we can check for it here too instead of just checking `/dev/sd`